.pre-commit-config.yaml

```diff
@@ -19,3 +19,5 @@ repos:
     rev: "v2.7.1"
     hooks:
       - id: prettier
+        types_or: ["javascript", "ts", "json"]
```

Comment:
Prettier should be applied to almost all files - its compatible with `.editorconfig` as well.

What was the context for restricting it down to just these files?

Reply:
I was seeing what patterns we had in other repos for consistencies sake, but I have removed it now.